sdk/keyvault/azure-security-keyvault-keys/src/private/package_version.hpp

```diff
@@ -20,6 +20,7 @@
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys { namespace _detail {
   /**
    * @brief Provides version information.
+   *
```

Comment:
I am guessing storage needs to make a similar change? Please share what you did to make it easier.

cc @Jinming-Hu

Reply:
use regex `/\*\*[\n\s]+.*[\n\s]+\*/` to find all cases, then add one line :)